src/Compilers/CSharp/Portable/Symbols/Source/SourceUserDefinedOperatorSymbolBase.cs

```diff
@@ -163,7 +169,15 @@ protected static DeclarationModifiers MakeDeclarationModifiers(MethodKind method
                         var requiredVersionArgument = new CSharpRequiredLanguageVersion(requiredVersion);
                         var availableVersionArgument = availableVersion.ToDisplayString();
 
-                        reportModifierIfPresent(result, DeclarationModifiers.Abstract, location, diagnostics, requiredVersionArgument, availableVersionArgument);
+                        if ((result & DeclarationModifiers.Abstract) != 0)
```

Comment:
>if ((result & DeclarationModifiers.Abstract) != 0) [](http://example.com/codeflow?start=24&length=50)

Why check this rather than calling `reportModifierIfPresent()` unconditionally, for `DeclarationModifiers.Abstract` and `DeclarationModifiers.Sealed`?

Reply:
> Why check this rather than calling 

Suppresses extra diagnostics for "abstract virtual" case

Reply:
I think there is a dedicated error reported for this case elsewhere.